test/functionalTest/cases/4113_alarm_for_notification_queue_overpassing_a_given_threshold/alarm_for_notification_queue_overpassing_a_given_threshold.test

```diff
@@ -0,0 +1,292 @@
+done
+echo
+echo
+
+
+echo "06. Create/update entity in serv2 4 times"
+echo "========================================="
+for i in {1..4}
+do
+   echo $i
+   orionCurl --url '/v2/entities?options=upsert,forcedUpdate' --payload "$payload" --tenant serv2 > /dev/null
+done
+echo
+echo
+
+
+echo "07. Create/update entity in serv3 5 times"
+echo "========================================="
+for i in {1..5}
+do
+   echo $i
+   orionCurl --url '/v2/entities?options=upsert,forcedUpdate' --payload "$payload" --tenant serv3 > /dev/null
+done
+echo
+echo
+
+
+echo "08. Create/update entity in serv4 5 times (default queue)"
+echo "========================================================="
+for i in {1..5}
+do
+   echo $i
+   orionCurl --url '/v2/entities?options=upsert,forcedUpdate' --payload "$payload" --tenant serv4 > /dev/null
+done
+echo
+echo
+
+
+echo "09. Grep log for notificationQueue alarm"
+echo "========================================"
+egrep 'Raising alarm|Releasing alarm' /tmp/contextBroker.log | awk -F'|' '{ print $10 }' | awk -F= '{ print $2 }'
+echo
+echo
+
+
+accumulatorStop $LISTENER_PORT
+
+
+--REGEXPECT--
+01. Subscribe serv1 to the accumulator endpoint that never responses
+====================================================================
+HTTP/1.1 201 Created
+Date: REGEX(.*)
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Location: /v2/subscriptions/REGEX([0-9a-f]{24})
+Content-Length: 0
+
+
+
+02. Subscribe serv2 to the accumulator endpoint that never responses
+====================================================================
+HTTP/1.1 201 Created
+Date: REGEX(.*)
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Location: /v2/subscriptions/REGEX([0-9a-f]{24})
+Content-Length: 0
+
+
+
+03. Subscribe serv3 to the accumulator endpoint that never responses
+====================================================================
+HTTP/1.1 201 Created
+Date: REGEX(.*)
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Location: /v2/subscriptions/REGEX([0-9a-f]{24})
+Content-Length: 0
+
+
+
+04. Subscribe serv3 to the accumulator endpoint that never responses
+====================================================================
+HTTP/1.1 201 Created
+Date: REGEX(.*)
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Location: /v2/subscriptions/REGEX([0-9a-f]{24})
+Content-Length: 0
+
+
+
+05. Create/update entity in serv1 3 times
+=========================================
+1
+2
+3
+
+
+06. Create/update entity in serv2 4 times
+=========================================
+1
+2
+3
+4
+
+
+07. Create/update entity in serv3 5 times
+=========================================
+1
+2
+3
+4
+5
+
+
+08. Create/update entity in serv4 5 times (default queue)
+=========================================================
+1
+2
+3
+4
+5
+
+
+09. Grep log for notificationQueue alarm
+========================================
+Raising alarm NotificationQueue serv1: notification queue reached maximum threshold
+Releasing alarm NotificationQueue serv1
```

Comment:
Why releasing alarm messages are appearing here if the endpoint never response during this test? Maybe something needs to be fixed in the implementation?

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/4375/commits/2e046041b810bc0c7791e35bafdf154ac2118df5